trunkbot/cmd/convo221.rb

```diff
@@ -1,11 +1,12 @@
 #!/usr/bin/env ruby
 
-n1 = ARGV[0] 
+root_path = File.expand_path('../..', File.dirname(__FILE__))
+
+n1 = ARGV[0]
 n2 = ARGV[1]
 
 begin
 
-  
   raise 'wrong argument count' if ARGV.length != 2
 
   ty = [ 'eight_ender', 'ol_qwerty_bastrd', 'HAM_RADIO', 'tyler' ]
```

Comment:
In absence of a DB some sort of centralized class for nicknames might not be a bad idea so you don't need to keep defining these